apps/src/templates/export/expo/App.js.ejs

```diff
@@ -106,14 +107,22 @@ export default class App extends React.Component {
       );
     }
 
+    const showWebView = !!height && indexUri;
+
+    if (showWebView && needsEmptyRender) {
+      // Workaround for react-native-webview issue with Android 10
+      // See https://github.com/react-native-community/react-native-webview/issues/656#issuecomment-551312436
+      this.setState({needsEmptyRender: false});
```

Comment:
Was going to say something about `setState` in `render` being an antipattern, but it looks like you've considered this and need to do it because showing the WebView isn't always guaranteed.

It might be cleaner to put a thin wrapper around the WebView just to address this bug, instead of mixing it with your app logic.

```jsx
class WebViewAndroid10Fix extends React.Component {
  static propTypes = WebView.propTypes;

  componentDidMount() {
    this.setState({renderedOnce: true});
  }

  render() {
    return (
      <WebView
        {...props}
        source={this.state.renderedOnce ? this.props.source : undefined}
      />
    );
  }
};
```

Reply:
I like this idea, but I'm hopeful that we'll be able to remove it quickly if the underlying bug is fixed.